ci/cron/daily-compat.yml

```diff
@@ -212,7 +212,7 @@ jobs:
           --logging.level.com.synopsys.integration=DEBUG \
           --detect.tools=DETECTOR \
           --detect.included.detector.types=YARN,NPM,CLANG \
-          --detect.npm.dependency.types.excluded=NON_PRODUCTION \
+          --detect.npm.dependency.types.excluded=DEV \
```

Comment:
do you know why yarn did not change? afaik the notion of dev dependencies is the same for both.

Reply:
The issue here is that Yarn did change, but NPM didn't, and in #14995 I updated both to match Yarn. Because the output mentions two errors, I saw 2 uses, and didn't think further.

Looking at it closer now, it just mentions the same error twice, so that's on me.

And, no, I have no idea why the Yarn side has been changed and the NPM one hasn't.

Reply:
thanks for the explanation, weird choice to switch over yarn but not npm but :shrug: